test/utils/nrosched/nrosched.go

```diff
@@ -92,10 +92,10 @@ func CheckPODWasScheduledWith(k8sCli *kubernetes.Clientset, podNamespace, podNam
 	return checkPODEvents(k8sCli, podNamespace, podName, schedulerName, isScheduledWith)
 }
 
-func CheckNROSchedulerAvailable(cli client.Client, nroSchedName string) *nropv1alpha1.NUMAResourcesScheduler {
+func CheckNROSchedulerAvailable(cli client.Client, NUMAResourcesSchedObjName string) *nropv1alpha1.NUMAResourcesScheduler {
```

Comment:
Would you prefer `nroSchedObjName` here? I was trying to capture NUMAResourcesScheduler object name explicitly to prevent anyone confusing it with NUMAResourcesOperator object name.

Reply:
let's try this name for some time and let's see how we feel using it for a while